mlperf_logging/mllog/constants.py

```diff
@@ -103,9 +103,13 @@
 OPT_LR_WARMUP_FACTOR = "opt_learning_rate_warmup_factor"
 OPT_LR_WARMUP_STEPS = "opt_learning_rate_warmup_steps"
 OPT_WEIGHT_DECAY = "opt_weight_decay"
+MIN_IMAGE_SIZE = "minimum image size"
```

Comment:
the value after "=" needs to match that in the spec doc (its not a description but actually being printed), e.g. this should be MIN_IMAGE_SIZE = "min_image_size"

Reply:
fixed as suggested. Thanks.